hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -259,6 +259,39 @@
         - 'kubernetes-e2e-{suffix}'
  
 - project:
+    name: kubernetes-e2e-gke-1-2
+    trigger-job: 'kubernetes-build-1.2'
+    test-owner: 'GKE on-call'
+    provider-env: |
+        {gke-provider-env}
+        export JENKINS_PUBLISHED_VERSION="ci/latest-1.2"
+    suffix:
+        - 'gke-release-1.2':
+            description: 'Run E2E tests on GKE from the release-1.2 branch.'
+            timeout: 30
+            job-env: |
+                export PROJECT="k8s-jkns-e2e-gke-1-2"
+                export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+                export GINKGO_PARALLEL="y"
+        - 'gke-serial-1.2':
```

Comment:
I did, but on second thought it's better to keep it consistent with the gce jobs.